onnxruntime/core/providers/dnnl/graph/graph_builder/dnnl_biasgelu_builder.cc

```diff
@@ -0,0 +1,62 @@
+#include "dnnl_graph_builder.h"
+
+namespace onnxruntime::ort_dnnl {
+
+enum InputTensors_BiasGelu : int {
+  IN_A = 0,
+  IN_B = 1,
+
+};
+
+enum OutputTensors_BiasGelu : int {
+  OUT_C = 0,
+};
+
+void DnnlGraphBuilder::BuildBiasGelu(const Node* node,
+                                          DnnlTensorPtrVector& inputs,
+                                          DnnlTensorPtrVector& outputs) {
+  // Generate the container
+  auto onnx_op = UniqueVectorOfUniqueNodes();
+
+  // Get names
+  auto node_name = node->Name();
+  auto bias_name = node_name + "_Decomposed_0_Bias";
+  auto gelu_name = node_name + "_Decomposed_1_GeLU";
+  std::string x_tensor_name;
+
+  {
+    // Create the bias I/O
+    auto bias_out_name = bias_name + "_out";
+    auto bias_inputs = std::make_shared<std::vector<DnnlTensorPtr*>>();
+    auto bias_outputs = std::make_shared<std::vector<DnnlTensorPtr*>>();
+    // Get Input tensors
+    auto src_0_tensor = *(inputs->at(IN_A));
+    auto src_1_tensor = *(inputs->at(IN_B));
```

Comment:
can these call New_DnnlNode::GetInput(NodeIndex input_idx).  It seems like calling the code like this is error prone.